packages/atomic/.storybook/manager.tsx

```diff
@@ -12,3 +12,41 @@ addons.register('SELECT-FIRST-STORY-BY-DEFAULT-ONCE', (api) => {
     api.selectStory(undefined, 'default');
   });
 });
+
+const observeAndExpandButtons = () => {
+  const observer = new MutationObserver(() => {
+    const buttonsToExpand = document.querySelectorAll(
+      'button[data-action="expand-all"][data-expanded="false"]'
+    );
+    if (buttonsToExpand.length > 0) {
+      buttonsToExpand.forEach((button) =>
+        (button as HTMLButtonElement).click()
+      );
+      observer.disconnect();
+    }
+  });
+
+  observer.observe(document.body, {
+    childList: true,
+    subtree: true,
+  });
+};
+
+const addNoIndexMetaTag = () => {
+  const metaTag = document.createElement('meta');
+  metaTag.name = 'robots';
+  metaTag.content = 'noindex';
+  document.head.appendChild(metaTag);
+};
+
+addons.register('expand-all-folders-on-intro', () => {
+  addons.getChannel().on(STORY_RENDERED, (storyId) => {
+    if (storyId === 'introduction--crawling') {
+      observeAndExpandButtons();
+      addNoIndexMetaTag();
+    }
+    if (storyId === 'introduction--default') {
+      addNoIndexMetaTag();
```

Comment:
Not a huge deal, but in effect, this is getting called every time you navigate back to the `introduction--default` story, creating a new noindex meta tag each time.

Reply:
Thanks; fixed!